lib/src/tools/over_react_format_tool.dart

```diff
@@ -21,20 +20,11 @@ class OverReactFormatTool extends DevTool {
   FutureOr<int> run([DevToolExecutionContext context]) async {
     Iterable<String> paths = context?.argResults?.rest;
     if (paths?.isEmpty ?? true) {
-      if (context != null) {
-        context.usageException(
-            '"hackFastFormat" must specify targets to format.\n'
-            'hackFastFormat should only be used to format specific files. '
-            'Running the command over an entire project may format files that '
-            'would be excluded using the standard "format" command.');
-      } else {
-        logError(
-            '"hackFastFormat" must specify targets to format.\n',
-            'hackFastFormat should only be used to format specific files. '
-                'Running the command over an entire project may format files that '
-                'would be excluded using the standard "format" command.');
-        return ExitCode.usage.code;
-      }
+      context.usageException(
```

Comment:
Sorry, I missed this when you addressed my previous comment. You're right that `context` is optional here, so we should still be protecting against a `null` value:
```suggestion
      context?.usageException(
```

I was just explaining that you don't need to implement a separate logging mechanism for when it is null because the only time that can happen is during unit tests or when someone instantiates this tool and runs it manually (which won't happen since it isn't public).

Reply:
Ohh yep, that makes sense haha. Awesome, updated that!!